tasks/update-versions.js

```diff
@@ -18,3 +18,7 @@ for (var index = 0, length = entries.length; index < length; index++) {
 }
 
 fs.writeFileSync('snapshots/versions.json', JSON.stringify(versions), { encoding: 'utf8' });
+
+var divshot = require('../divshot.json');
```

Comment:
In general, I think you can leave the `.json` off when `require`ing (though it should work either way).